schema/workflow/unit/io/download_from_object_storage.json

```diff
@@ -0,0 +1,62 @@
+{
+    "schemaId": "workflow-unit-download-from-object-storage",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "download-from-object-storage io schema",
+    "type": "object",
+    "allof": [
+        {
+            "$ref": "../../unit.json"
+        }
+    ],
+    "properties": {
+        "filename": {
+            "description": "name given to the file when it is downloaded",
+            "type": "string"
+        },
+        "objectData": {
+            "description": "information describing the file and where it was run",
+            "type": "object",
+            "properties": {
+                "CONTAINER": {
+                    "description": "Container that ran the job resulting in the file",
+                    "type": "string"
+                },
+                "NAME": {
+                    "description": "Path to where the job put the file",
+                    "type": "string"
+                },
+                "PROVIDER": {
+                    "description": "Provider of the computational resources",
+                    "type": "string"
+                },
+                "REGION": {
+                    "description": "Region of the cloud provider's server",
```

Comment:
This is specifically the region for the object container specified in CONTAINER.

Reply:
Description here has been fixed